third_party/move/move-vm/types/src/loaded_data/runtime_types.rs

```diff
@@ -113,12 +114,14 @@ impl DepthFormula {
 
 #[derive(Debug, Clone, Eq, Hash, PartialEq)]
 pub struct StructType {
+    pub idx: StructNameIndex,
     pub fields: Vec<Type>,
     pub field_names: Vec<Identifier>,
     pub phantom_ty_args_mask: SmallBitVec,
     pub abilities: AbilitySet,
     pub type_parameters: Vec<StructTypeParameter>,
-    pub name: Arc<StructIdentifier>,
+    pub name: Identifier,
+    pub module: ModuleId,
```

Comment:
where is this used in the new code?

Reply:
This is used in transaction argument validation IIRC.